BTSPocket/Networking/Constants.swift

```diff
@@ -20,8 +21,8 @@ struct Constants {
     struct Endpoints {
         static let postAuthentication = "https://platform.bluetrail.software/api/users/login"
```

Comment:
you already have `serverAddress` 
Avoid repeating same address on method names

replace  `"https://platform.bluetrail.software/api/users/login"` by `"api/users/login"`